### DISCUSSION TITLE
Set up job failing in GitHub Actions

### DISCUSSION THREAD

**Discussion by mrdailey99** [community/community#57946] [Actions] [answered]
### Select Topic Area

Question

### Body

I have a GitHub Workflow that was running seamlessly until today (in the past few months, I've had almost no failures). Today, I started to see a strange failure, seemingly related to the Set Up job step, which is a first.
<img width="766" alt="image" src="https://github.com/community/community/assets/49916244/d46acf27-e45a-42d3-8804-fbb1f973d42d">
I have tried modifying all of the versions of my actions to see which one is the culprit, and I believe it is the Upload Artifact Action. I've tried:
```

          uses: actions/upload-artifact@v2.2.1
          uses: actions/upload-artifact@v3
          uses: actions/upload-artifact@v3.1.2
          uses: actions/upload-artifact@v3.0.0
```
I did not make any changes to my workflow today, and just saw this failure all of a sudden on one of my jobs. 
Every run is the same error.

Error: An action could not be found at the URI 'https://api.github.com/repos/wei/wget/tarball/ca0a6b7f0fdd7ffbe9d4cec0b9696f2dc658d3c9'

Any ideas?
Public repo failure: https://github.com/mrdailey99/provardx/actions/runs/5256407081/jobs/9497689490

**Comment by mrdailey99** [comment]
Forgot to mention I have also tried commenting out the upload-artifact action, so that might not even be the issue.

**Comment by airtower-luna** [comment, marked as answer]
The [error message](https://github.com/mrdailey99/provardx/actions/runs/5256407081/jobs/9497689490#step:1:40) indicates that downloading the `wei/wget` action failed. And indeed neither the repository nor the user seem to exist. I can't say why they disappeared, but that's pretty clearly the problem. :sweat_smile: 

It's probably best to just call `wget` or `curl` directly in a `run` step. :slightly_smiling_face:

**Reply by mrdailey99** [reply]
Ah... that makes sense now. I previously used the action as a wrapper because invoking wget directly didn't seem to be working on ubuntu-latest. I can try that instead, thanks!
